api/forms.py

```diff
@@ -0,0 +1,82 @@
+# encoding: utf-8
+from django import forms
+from django.forms.widgets import (TextInput, Textarea, RadioSelect, CheckboxSelectMultiple,
+                                 MultiWidget)
+from crispy_forms.helper import FormHelper
+from crispy_forms.utils import render_crispy_form
+
+
+class HopperForm(forms.Form):
+    def __init__(self, *args, **kwargs):
+        model = kwargs.pop('model', None)
+        elements = model.convert_to_dict(model.elements)
+        super(HopperForm, self).__init__(*args, **kwargs)
+        self.type_widget_mapping = {
+            'input': TextInput,
+            'fieldset': MultiWidget,
+            'textarea': Textarea,
+            'radio': RadioSelect,
+            'checkbox': CheckboxSelectMultiple,
+        }
+        self.helper = FormHelper()
+        self.helper.form_class = model.css_classes
+        self.helper.form_action = model.action
+        self.helper.form_method = model.method
+        self.helper.field_class = model.elements_css_classes
+        self.helper.form_show_labels = False
```

Comment:
Why not? How should the enduser, who uses the prerendered html know what field stands for what?